frontend/src/components/custom-tools/settings-modal/SettingsModal.jsx

```diff
@@ -65,7 +65,7 @@ function SettingsModal({ open, setOpen, handleUpdateTool }) {
       items.splice(
         position,
         0,
-        getMenuItem("Summary Manager", 2, <FileTextOutlined />)
+        getMenuItem("SummarizedExtraction", 2, <FileTextOutlined />)
```

Comment:
@jagadeeswaran-zipstack why do we use camel case here? Please add screenshots in the PR as well if possible

Reply:
@shuveb asked to use camel case here. Sure, I'll add screenshots in the PR